pdcupdater/handlers/modules.py

```diff
@@ -75,6 +75,12 @@ def handle(self, pdc, msg):
 
         unreleased_variant = self.get_or_create_unreleased_variant(pdc, body)
 
+        if body['state'] == 5:
+            uid = unreleased_variant['variant_uid']
+            # This submits an HTTP PATCH.
+            # The '/' is necessary to avoid losing the body in a 301.
+            pdc['unreleasedvariants'][uid + '/'] += {'variant_uid': uid, 'active': True}
```

Comment:
I'm wondering why the `variant_uid` is set here. I'm assuming that the uid is already correct since you seem to be querying by it with `pdc['unreleasedvariants'][uid + '/']`. I might be just misunderstanding what beanbag does in this instance though.